server/lib/providers/tfs-tfvc.js

```diff
@@ -166,17 +166,17 @@ const downloadFile = (file, changesetId) => {
   const auth = new Buffer(`${config('USERNAME')}:${config('TOKEN')}`).toString('base64');
 
   const options = {
+    method: 'GET',
     headers: {
-      Authorization: `Basic ${auth}`,
-      'Content-Type': 'text/html'
+      Authorization: `Basic ${auth}`
     },
-    uri: `https://${config('INSTANCE')}.visualstudio.com/${config('COLLECTION')}/_apis/tfvc/items?path=${file.path}${versionString}&api-version=1.0`
+    url: `https://${config('INSTANCE')}.visualstudio.com/${config('COLLECTION')}/_apis/tfvc/items?path=${file.path}${versionString}&api-version=5.0&includeContent=true`
   };
 
-  return request.get(options)
-    .then((data) => ({
+  return axios(options)
+    .then((response) => ({
       fileName: file.path,
-      contents: data
+      contents: response.data && response.data.content
     }))
     .catch(e => e);
```

Comment:
Calls like this are no-ops; it might be better to omit them so it doesn't seem like the error is being handled.